src/components/ReplyForm/ReplyForm.tsx

```diff
@@ -36,20 +50,28 @@ const ReplyForm = ({ errors, isPending, onCancel, onChange, setFieldValue, value
         onChange(getFormattedCommentText(nextEditorState).text);
         setFieldValue('editorState', nextEditorState);
     };
-    const handleKeyDown = (event: React.KeyboardEvent): void => {
-        if (event.key !== KEYS.escape) {
-            return;
+
+    // The entire annotation mode keydown event listener is attached to real dom's document
+    // Have to attach this event listener to real dom too, in order to capture the keydown event
+    // before annotation mode event listner captures it
+    React.useEffect(() => {
+        const { current: formEl } = formRef;
+
+        if (!formEl) {
+            return noop;
```

Comment:
Is this needed? It seems like we would just wrap the code in an `if (formEl)` block in this case.